packages/mdx-embed/src/components/vimeo/vimeo.tsx

```diff
@@ -6,13 +6,13 @@ export interface IVimeoProps {
   /** Vimeo id */
   vimeoId: string;
   /** Skip to a time in the video */
-  skipTo: {
+  skipTo?: {
```

Comment:
Nice catch, this and `autoPlay` should be optional. It's one small downside to using `.mdx` for the Stories... no type checking!

Reply:
Yes, indeed. :) Gotta love them types 👍  Thanks for taking a look!